faker/compat.py

```diff
@@ -1,6 +1,6 @@
 # Python 2/3 compat
 import sys
-if sys.version < '3':
+if sys.version_info[0] < 3:
```

Comment:
The pull request is about a year old now; I'm not sure I should make that change when it makes little difference.